charts/plugin-fees/templates/bootstrap-mongodb.yaml

```
@@ -0,0 +1,104 @@
+{{- if .Values.global.externalMongoDefinitions.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "plugin-fees.fullname" . }}-bootstrap-mongodb
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)